### PR TITLE
kv/RocksDBStore: Reduce duplicate code

### PR DESCRIPTION
**Pull request by shinobu-x**
Signed-off-by: Shinobu Kinjo <shinobu@redhat.com>

### CODE REVIEW

File: src/kv/RocksDBStore.cc [hunk 1]
```diff
@@ -756,32 +756,43 @@ int RocksDBStore::submit_common(rocksdb::WriteOptions& woptions, KeyValueDB::Tra
   return s.ok() ? 0 : -1;
 }
 
-int RocksDBStore::submit_transaction(KeyValueDB::Transaction t) 
-{
+void RocksDBStore::_submit_transaction(KeyValueDB::Transaction& t,
```
Comment (ifed01) [inline src/kv/RocksDBStore.cc]: No need to pass lat parameter, it's used inside the function only

Comment (ifed01) [inline src/kv/RocksDBStore.cc]: You may return result in a regular manner via function return, no need for passing it by ref

Comment (ifed01) [inline src/kv/RocksDBStore.cc]: And perhaps there is no much sense in this new function - you can modify submit_common. Strictly IMO.

**Comment by shinobu-x** [comment]
@ifed01 like now?

**Comment by jdurgin** [comment]
closing since it needs rebase and is just a cleanup.

**Pull request closed without merging**